### PR TITLE
Use network models, rather than local ones

### PR DESCRIPTION
**Pull request by mruwnik**
Removes the local models and uses the deployed microservice model servers

### CODE REVIEW

File: src/stampy_nlp/models.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+
+class Model:
+    def __init__(self, host, model_name, pinecone=None):
+        self.host = host
+        self.pinecone = pinecone or get_index()
+        self.model_name = model_name
+
+    def _post(self, endpoint, payload):
+        resp = requests.post(self.host + endpoint, json=payload)
```
[the remark this answers is not in the document]
Comment (ccstan99) [reply src/stampy_nlp/models.py:24]: Ultimately, the deployed models should only be accessible by our own deployed server. You're probably more efficient than I am at hunting this down, but there should be an option to set the firewall rule to only allow incoming connections from specific ip(s) or within the same GCP project. Somewhere here? 
https://cloud.google.com/vpc/docs/firewalls
https://cloud.google.com/run/docs/securing/ingress

**Review by ccstan99 — COMMENTED** [review]
You might want to kill me but... Can we support using local models in a dev env? Especially for encoding docs and inserting into pinecone? Those tasks are very quick on a local GPU but are really time-consuming on a CPU. 

Also, I'm a little disoriented now. Did entire directory `encode-faq-py` get removed in PR#5?! Sorry, we still need `encode-extracted-chunks.py` and `encode-lit-abstracts.py` to create embeddings for the other apps. They were only getting called manually. I guess I can just keep them locally until it gets integrated with other work? 

Also for the local cron job, I am using `encode-faq-titles.py` without hitting the endpoint.

**Comment by mruwnik** [comment]
> You might want to kill me but... Can we support using local models in a dev env? Especially for encoding docs and inserting into pinecone? Those tasks are very quick on a local GPU but are really time-consuming on a CPU.
> 
> Also, I'm a little disoriented now. Did entire directory `encode-faq-py` get removed in PR#5?! Sorry, we still need `encode-extracted-chunks.py` and `encode-lit-abstracts.py` to create embeddings for the other apps. They were only getting called manually. I guess I can just keep them locally until it gets integrated with other work?
> 
> Also for the local cron job, I am using `encode-faq-titles.py` without hitting the endpoint.

You can manually run specific model_server instances with different ports, e.g.

* retriever:  `MODEL_NAME=sentence-transformers/multi-qa-mpnet-base-cos-v1 MODEL_TYPE=encoder uvicorn model_server.app:app --port 8123 --reload`
* qa: `MODEL_NAME=deepset/electra-base-squad2 MODEL_TYPE=pipeline model_server.app:app --port 8124 --reload`
* lit search: `MODEL_NAME=allenai/specter MODEL_TYPE=encoder uvicorn model_server.app:app --port 8125 --reload`

**Review by ccstan99 — APPROVED** [review]
Thanks! I've made some changes to the encodings to pinecone code. So I can "resurrect" that folder with the current working files. I'll play around with your cleaned up and merge it with other stuff I've been doing.

**Pull request merged**